src/react/components/common/assetPreview/assetPreview.tsx

```diff
@@ -10,6 +10,8 @@ export type ContentSource = HTMLImageElement | HTMLVideoElement;
  * AssetPreview component properties
  */
 export interface IAssetProps {
+    /** The current project */
+    project: IProject;
```

Comment:
Instead of expected the whole object here I would like to scope this down a bit so only the asset level properties relevant for the asset preview.  @PIC123 has something in his current PR which should map to the properties that you need.

Reply:
I'm fine with whatever limited scope is best. @PIC123 and my PR have some duplicates we should probably reconcile before the final merge

Reply:
Updated to use additional settings